README.md

```diff
@@ -32,172 +13,16 @@ Supported cloud providers:
-* [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/)
-
-## Deployment Options
-
-Hive contains an operator which is responsible for handling deployment logic for the rest of the components. In the near future this operator may be installable via OLM.
-
-### Deploy Hive Operator Using Latest Master Images
-
-To deploy the operator from a git checkout:
-
-  `$ make deploy`
-
-By default the operator will use the latest images published by CI from the master branch.
-
-You should now see hive-operator, hive-controllers, and hiveadmission pods running in the hive namespace.
-
-### Deploy Hive Operator Using Custom Images
-
- 1. Build and publish a custom Hive image from your current working dir: `$ IMG=quay.io/dgoodwin/hive:latest make buildah-push`
- 2. Deploy with your custom image: `$ DEPLOY_IMAGE=quay.io/dgoodwin/hive:latest make deploy`
-
-### Deploy Hive via OLM
-
-We do not currently publish an official OLM operator package, but you can run or work off the test script below to generate a ClusterServiceVersion, OLM bundle+package, registry image, catalog source, and subscription.
-
-`$ REGISTRY_IMG="quay.io/dgoodwin/hive-registry" DEPLOY_IMG="quay.io/dgoodwin/hive:latest" hack/olm-registry-deploy.sh`
-
-
-### Run Hive Operator From Source
-
-NOTE: assumes you have previously deployed using one of the above methods.
-
- 1. `$ oc scale -n hive deployment.v1.apps/hive-operator --replicas=0`
- 1. `$ make run-operator`
-
-### Run Hive From Source
-
-NOTE: assumes you have previously deployed using one of the above methods.
-
- 1. `$ oc scale -n hive deployment.v1.apps/hive-controllers --replicas=0`
- 1. `$ make run`
-
-## Using Hive
-
-1. Obtain a pull secret from try.openshift.com and place in your home directory at `$HOME/.pull-secret`.
-1. Ensure that you are logged in to AWS via the AWS CLI or that your AWS credentials are set in environment variables: AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY
-1. Note the use of an SSH key below to access the instances if necessary. (this should typically not be required)
-
-### Create a ClusterDeployment using the latest OpenShift release and installer image
-
-Hiveutil offers a create-cluster subcommand which simplifies creating the API custom resources for a Hive ClusterDeployment.
-
-```bash
-make hiveutil
-```
-
-
-```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com mycluster
-```
-
 To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
 By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--hive-image` can be specified to use a specific Hive image to run the install, and `--release-image` can be specified to control which OpenShift release image to install in the cluster.
```

Comment:
This needs to be moved to `Installing Hive` section.

Reply:
This section is about use of create-cluster utility, it seems like it is correct here, this is how you use hive, not install it.

Reply:
@dgoodwin I am confused. The text says `To view what create-cluster generates, without submitting it to the API server, add -o yaml to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with oc apply` 

But there is no create-cluster command before it, so there is a disconnect for sure. 
![hive_docs_review](https://user-images.githubusercontent.com/3125299/60169959-6065b180-9825-11e9-8782-4e99305193f0.png)

Reply:
Oh there's a problem with the command quoting! I see now. Working on it.

Reply:
Oh no still confused, if I go to view file, from the changes tab in the PR, I can see the quoted command, but it is not present in your screenshot.

Reply:
I see the rendered changes by clicking the top right corner `view doc`

![doc_review_temp](https://user-images.githubusercontent.com/3125299/60178391-15559980-9839-11e9-841b-8187cec2de42.png)

Reply:
Also  I can see command is getting removed in the diff (github UI).